torch/_inductor/codecache.py

```diff
@@ -1186,6 +1187,254 @@ def load(cls, kernel_name, source_code):
         return getattr(mod, kernel_name)
 
 
+def _cuda_compiler() -> str:
+    return "nvcc"
```

Comment:
Maybe we should use the same nvcc executable that is used and detected by torch's CMAKE build script, as that might not be the one that's on the path.

Reply:
Do you refer to `cmake/public/cuda.cmake`? From my understanding, this is written in CMake language and requires cmake to execute. I'm not familiar with CMake, and it doesn't look very trivial to do the integration. Since the cutlass integration is new, I'd like make it work first and then update the surrounding pieces, e.g. env setup, etc. For now I would just add a config in Inductor to let user specify nvcc path if they need.

Reply:
This won't work since CMAKE often runs on a different machine than pytorch gets `pip installed` on.